utils/streams.js

```diff
@@ -0,0 +1,156 @@
+import commander from 'commander';
+import through from 'through2';
+import fs from 'fs';
+import path from 'path';
+import { promisify } from 'util';
+import csvjson from 'csvjson';
+// import https from 'https';
+const readFileAsync = promisify(fs.readFile);
+const readDirAsync = promisify(fs.readdir);
+
+function streams() {
+    commander.option('-a, --action <string>', 'action name taken')
+        .option('-f, --file <string>', 'file will read')
+        .option('-p, --path <string>', 'path for css folder')
+        .parse(process.argv);
+    executeTheAction();
+}
+function executeTheAction() {
```

Comment:
The homework assignment doesn't define the way how to handle multiple actions, anyway switch statement has its right to exist. But the best practice is to avoid switch statements in your code. For such cases you can create an object with properties that are functions. And then you can access these functions with bracket notation.

`const functions =  {
  reverse,
  transform,
  outputFile
}`

`functions[commander.action](arguments)`

where arguments should be appropriate for the particular case, and of course `reverse`, `transform` etc. should be define in your code (actually they are)

Reply:
Thanks for the suggestion. It feels it's a better approached to use the suggested method. 
Updated the code to use Object of functions and call the appropriate action accordingly.